tests/unit/scripting.tcl

```diff
@@ -1231,6 +1231,9 @@ start_server {tags {"scripting needs:debug"}} {
 
     for {set i 2} {$i <= 3} {incr i} {
         for {set client_proto 2} {$client_proto <= 3} {incr client_proto} {
+            if {[lsearch $::denytags "resp3"] >= 0} {
+                if {$i == 3 || $client_proto == 3} {continue}
```

Comment:
are you sure we need the `$i == 3` part?
it tests the script engine, no the client protocol part...

Reply:
you're right. fixed in [38e0c42](https://github.com/redis/redis/pull/11090/commits/38e0c42d59b27fa105085d487a35488d5aff6c3d)